client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-preview.tsx

```diff
@@ -50,6 +51,14 @@ const PatternAssemblerPreview = ( { header, sections = [], footer }: Props ) =>
 		setWebPreviewFrameContainer( document.querySelector( '.web-preview__frame-wrapper' ) );
 	}, [] );
 
+	useEffect( () => {
+		setScrollToSelector( null );
```

Comment:
Do you think it will be helpful to scroll to a selected section, especially on a long preview content?

Reply:
I'm not sure what you mean. This line is avoiding the scroll in the iframe when the footer has not changed.

Reply:
We want to explore how to scroll to a specific section in another PR. The problem is that the sections don't have an ID or some specific selector to use as an anchor. I'll create another issue to explore ideas.